session/session.go

```diff
@@ -2746,15 +2746,19 @@ func InitDDLJobTables(store kv.Storage) error {
 		if err != nil {
 			return err
 		}
-		p := parser.New()
-		for _, tbl := range DDLJobTables {
-			id, err := t.GetGlobalID()
-			if err != nil {
-				return errors.Trace(err)
+		if s, ok := store.(kv.SplittableStore); ok && atomic.LoadUint32(&ddl.EnableSplitTableRegion) == 1 {
```

Comment:
It's better to use a function to handle these tables, so that we don't need to write the same code for MDL table.

Reply:
Done, PTAL @wjhuang2016